src/auth/AuthSessionHandler.cc

```diff
@@ -38,14 +38,3 @@ AuthSessionHandler *get_auth_session_handler(CephContext *cct, int protocol, Cry
   }
   return NULL;
 }
-
-
-void AuthSessionHandler::print_auth_session_handler_stats() {
```

Comment:
@tchaikov 
Yes, that's right. So what do you suggest ? Remove or not remove ? I'm slightly inclined to "Not Remove".